doc/classes/TreeItem.xml

```diff
@@ -411,6 +418,20 @@
 				If [code]true[/code], the column [code]column[/code] is checked.
 			</description>
 		</method>
+		<method name="set_collapsed_recursive">
+			<return type="void">
+			</return>
+			<argument index="0" name="enable" type="bool">
+			</argument>
+			<argument index="1" name="ignore_active" type="bool">
+			</argument>
+			<argument index="2" name="ignore_self" type="bool">
+			</argument>
+			<description>
+				If [code]enable[/code] is [code]true[/code] collapses, else uncollapses, this TreeItem and all its descendants recursively, ignoring any active TreeItem if [code]ignore_active[/code] is [code]true[/code].
```

Comment:
This TreeItem and all its descendants collapse recursively when [code]enable[/code] is [code]true[/code], ignoring any active TreeItem if [code]ignore_active[/code] is [code]true[/code].

Reply:
I think this could be a bit confusing as someone might interpret the "enable" parameter to be a toggle for "recursive or not" instead of a toggle for "collapsed or not".

How about:
>This TreeItem and all its descendants recursively collapse when [code]enable[/code] is [code]true[/code], otherwise expanding, ignoring any active TreeItem if [code]ignore_active[/code] is [code]true[/code].